juniper_junos-formula/juniper_junos/files/routes.j2

```diff
@@ -0,0 +1,14 @@
+{#- FIXME: move these to context variables #}
```

Comment:
I do not see this one to be included .. since here we do only static route I guess is ok to be included in baseline.j2

Reply:
Nice observation, it seems I forgot to apply and test the routes template. Thanks, I'll try it there.

Reply:
Added via f68cc77e22cfed5dd4b87b9dad73716f5243b261.